src/huggingface_hub/hf_api.py

```diff
@@ -1646,6 +1647,11 @@ def _inner(self, *args, **kwargs):
     return _inner  # type: ignore
 
 
+class EndpointsScalingMetric(Enum):
```

Comment:
We could make `EndpointsScalingMetric` inherit from `str` to simplify usage and typing: that way, we can remove `str` from the type annotation and removes the need to cast with `EndpointsScalingMetric(...)`
```suggestion
class EndpointsScalingMetric(str, Enum):
```

Reply:
thanks. Btw I just dropped the class in this file is it ok ? Or you prefer me to store it in another module to keep things tidy ?